docs/download.md

```diff
@@ -98,7 +98,7 @@ Software Requirements:
 *Some hardware may have a minimum driver version greater than v450.80.02+.  Check the GPU spec sheet
 for your hardware's minimum driver version.
 
-*For Cloudera and EMR support, please refer to [FAQ section](./FAQ.md).
+*For Cloudera and EMR support, please refer to the [Distributions](./FAQ.md#which-distributions-are-supported) section of the FAQ.
```

Comment:
This needs to be applied to all sections of the download.html page that refer to Cloudera.

Reply:
Oh sorry done.